docs/babelconfigjs.md

```diff
@@ -0,0 +1,97 @@
+---
+title: babel.config.js
+id: babelconfigjs
+---
+
+## Lookup behavior
+
+The lookup behavior is direct as Babel will only look for a `babel.config.js` in the root directory (where your `package.json` is by default). If configuration is needed, see [specifying the root folder](babelconfigjs.md#specifying-the-root-folder).
+
+## Merging behavior
+
+Unlike other formats (see the [configuration guide](configuration.md)), Babel won't try to merge configurations.
+
+> Note: the configuration defined in `babel.config.js` is applied to your entire project.
+
+## Conditional configuration
+
+Since it's just a regular JavaScript file, you can write any arbitrary conditions before returning the configuration, for example:
+
+```js
+module.exports = function () {
+  const plugins = [ ... ];
+
+  if (true === false) {
```

Comment:
We should use something real here? Or just combine it with the environment example